src/decorator-annotator.ts

```diff
@@ -28,6 +28,38 @@ interface ConstructorParameter {
   decorators: ts.Decorator[]|null;
 }
 
+export function shouldLower(decorator: ts.Decorator, typeChecker: ts.TypeChecker) {
+  for (const d of getDecoratorDeclarations(decorator, typeChecker)) {
+    // Switch to the TS JSDoc parser in the future to avoid false positives here.
```

Comment:
While you're at it, should this be a TODO() comment?

Reply:
Done.